aptos-move/framework/aptos-framework/sources/aptos_account.move

```diff
@@ -199,7 +199,7 @@ module aptos_framework::aptos_account {
     /// This would create the recipient APT PFS first, which also registers it to receive APT, before transferring.
     /// TODO: once migration is complete, rename to just "transfer_only" and make it an entry function (for cheapest way
     /// to transfer APT) - if we want to allow APT PFS without account itself
-    fun fungible_transfer_only(
+    public(friend) entry fun fungible_transfer_only(
```

Comment:
this is so that I can call it as entry function, but also so I can remove it later - right @lightmark ?

Reply:
I think that's the backdoor @movekevin mentioned.